draft-ietf-privacypass-batched-tokens.md

```diff
@@ -428,14 +428,15 @@ verification works exactly as specified in {{RFC9578}}.
 # Generic Token Batch Issuance {#generic-batch}
 
 This section describes an issuance protocol mechanism for issuing multiple
-tokens in one round trip between Client and Issuer. An generic batch token
+tokens in one round trip between Client and Issuer. A generic batch token
 request can contain token requests for any token type.
 
 ## Client-to-Issuer Request {#generic-client-to-issuer-request}
 
 The Client first generates all of the individual TokenRequest structures that
 are intended to be batched together. This request creation follows the protocol
-describing issuance, such as {{RFC9578, Section 5.1}} or {{RFC9578, Section 6.1}}.
+describing issuance, such as {{!AUTHSCHEME=RFC9577, Section 5.1}} or {{RFC9578,
```

Comment:
why the change here? RFC8577 auth scheme does not define any issuance protocol. Section 5.1 of RFC 9578 does though